src/python/dxpy/cli/parsers.py

```diff
@@ -281,6 +281,10 @@ def __call__(self, parser, namespace, values, option_string=None):
                           help='If set to true, only projects that contain PHI data will be retrieved. ' +
                           'If set to false, only projects that do not contain PHI data will be retrieved.')
 
+database_ui_view_only = argparse.ArgumentParser(add_help=False)
```

Comment:
I don't think this needs flag needs its own parser. You can add `'--database-ui-view-only'` directly to `parser_new_project` like `parser_update_project` in `dx.py`